src/crawlers/crawler_utils.js

```diff
@@ -29,3 +30,34 @@ export function throwOnBlockedRequest(session, statusCode) {
         throw new Error(`Request blocked - received ${statusCode} status code.`);
     }
 }
+
+/**
+ * Merges multiple cookie strings. Keys are compared case-insensitively, the casing used
+ * on first appearance will be used.
```

Comment:
> Keys are compared case-insensitively

Cookie names are case sensitive. `test` and `TEST` are two different cookies.

Reply:
> it feels more like an error from user (that we could fix this way) rather than something done on purpose.

Mistyping the cookie name is an error from user as well. The spec exists to unify the behavior and we should follow it.

> It feels like we are doing a bit of autocorrect for such cases.

Exactly. I'm -1 on magical behavior. I see no reason why anybody would provide a name with incorrect casing on purpose.